src/olympia/files/tests/test_commands.py

```diff
@@ -58,3 +67,117 @@ def test_force_extract(self):
         file_ = File.objects.no_cache().get(id=file_.id)
         assert WebextPermission.objects.get(file=file_)
         assert len(file_.webext_permissions_list) == 5
+
+
+@override_settings(AMO_LANGUAGES=('fr', 'de', 'zh-CN'))
```

Comment:
Add one locale for which an error is returned. See also above regarding exceptions and http errors